starter-core/src/main/java/io/micronaut/starter/feature/build/maven/templates/pom.rocker.raw

```diff
@@ -276,16 +276,6 @@ MavenBuild mavenBuild
                  <artifactId>@processor.getArtifactId()</artifactId>
 @if (processor.getVersion() != null) {
                  <version>@processor.getVersion()</version>
-}
-@if (processor.getExclusions() != null) {
-                 <exclusions>
```

Comment:
it was failing telling there is not exclusions for https://maven.apache.org/plugins/maven-compiler-plugin/apidocs/org/apache/maven/plugin/compiler/DependencyCoordinate.html